### PR TITLE
Prepare Manifesto and Key features sections for website

### PR DESCRIPTION
**Pull request by tgorgol**
**Description**

Changes proposed in this pull request:

- Prepare Manifesto and Key features sections for website

Preview of new components: https://github.com/kyma-project/website/pull/627

**Related issue(s)**

Manifesto: https://github.com/kyma-project/website/issues/624
Key features: https://github.com/kyma-project/website/issues/625

### CODE REVIEW

File: content/i18n/en/siteMetadata.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "title": "Kyma",
   "shortDescription": "An easy way to extend enterprise applications on Kubernetes.",
-  "description": "Kyma is an open-source project designed natively on Kubernetes. It allows you to extend enterprise applications in a quick and modern way, using serverless computing or microservice architecture.",
+  "description": "Kyma is a cloud-native application runtime that gives you the power of Kubernetes but without the hassle of worrying about security, configuration, and integration of tools.",
```
[the remark this answers is not in the document]
Comment (kazydek) [reply content/i18n/en/siteMetadata.json:4]: This description should reflect our manifesto - as you can see in [PR 495](https://github.com/kyma-project/community/pull/495/files), it has not been decided yet what these abstracts would look like. For now, I would stick to what you have above (and what was used in the design).

File: src/common/i18n/flattenMessages.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ const flattenMessages = (nestedMessages: any, prefix = "") =>
     const value = nestedMessages[key];
     const prefixedKey = prefix ? `${prefix}.${key}` : key;
 
-    if (typeof value === "string") {
+    if (typeof value === "string" || Array.isArray(value)) {
```
[the remark this answers is not in the document]
Comment (aerfio) [reply src/common/i18n/flattenMessages.ts]: I'd rather have proper types here, so while you're here could you somehow create proper types for this function? We need to get rid of that damn `any` in order to facilitate future work here

File: src/views/landingPage/Features/FeatureCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { FeaturesI18nTarget } from "@typings/landingPage";
+import { IntlInterface, injectIntl } from "@common/i18n";
+import H from "@components/shared/H";
+import { ImageWrapper } from "./styled";
+import Extensibility from "@views/landingPage/assets/landing-page/features/extensibility.svg";
+import Runtime from "@views/landingPage/assets/landing-page/features/runtime.svg";
+import Support from "@views/landingPage/assets/landing-page/features/support.svg";
```
Comment (aerfio) [inline src/views/landingPage/Features/FeatureCard/index.tsx:8]: change those imports to something like `extensibilitySVG`, as the proper practice is that in React we use capitalised imports for components

File: src/views/landingPage/Manifesto/index.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  ButtonWrapper,
+} from "./styled";
+import Button from "@components/shared/Button";
+import Link from "@components/shared/Link";
+import Icon from "@components/shared/Icon";
+import { scrollToAnchor } from "@common/utils/scrollToAnchor";
+
+const gt = getTranslation("landingPage.manifesto");
+
+const scrollToFeatures = (e: any) => {
```
Comment (aerfio) [inline src/views/landingPage/Manifesto/index.tsx:25]: React.MouseEvent<HTMLAnchorElement, MouseEvent> instead of any

**Review by aerfio — CHANGES_REQUESTED** [review]
Looks nice! Make sure you've minified svgs using svgo or some other tool (if they are mangled after this operation then just leave them as they are)

**Review by magicmatatjahu — APPROVED** [review]

File: src/views/landingPage/Features/Header/styled.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    text-align: center;
+  }
+
+  > div {
+    max-width: 567px;
+    margin: 0 auto;
+  }
+`;
+
+export const SpellingOfText = styled.pre`
```
Comment (valentinvieriu) [inline src/views/landingPage/Features/Header/styled.ts:17]: Can we use paragraph in here, just for SEO consistency?

**Comment by valentinvieriu** [comment]
Can we use paragraph and h2 without span? It's a small change, but for SEO consistency
image

File: content/i18n/en/siteMetadata.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (valentinvieriu) [reply content/i18n/en/siteMetadata.json]: Yes. The final text will be in a separate issue, unfortunately.

File: src/views/landingPage/Features/FeatureCard/index.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+const FeatureCardRaw: React.FunctionComponent<Props & IntlInterface> = ({
+  feature,
+  formatMessage,
+  formatArray,
+}) => (
+  <FeatureWrapper>
+    <ImageWrapper>
+      <img src={getImageForFeature(feature)} />
```
Comment (valentinvieriu) [inline src/views/landingPage/Features/FeatureCard/index.tsx:34]: Please add alt attribute. Google Lighthouse complains about this
image

File: src/views/landingPage/Manifesto/index.tsx [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+                  {formatMessage({ id: "brand" })}
+                </FormattedBrand>
+              ),
+            }}
+          />
+        </HeadlineWrapper>
+        <ButtonWrapper>
+          <Link.External to="https://github.com/kyma-project/kyma">
+            <Button.Normal iconName="github-alt" iconPrefix="fab">
+              <FormattedMessage id={gt("githubButton")} />
```
Comment (valentinvieriu) [inline src/views/landingPage/Manifesto/index.tsx:56]: It seems lighthouse complains here too
[Link-name](https://web.dev/link-name/?utm_source=lighthouse&utm_medium=devtools)
I believe we need to add some text like : "Scroll to next section" that is hidden

File: src/views/landingPage/Features/FeatureCard/index.tsx (hunk 6 above)
Comment (valentinvieriu) [reply src/views/landingPage/Features/FeatureCard/index.tsx:34]: can we also force a fixed width here. It seem sthe SVGs are not the same size
237 sounds like a round number :)
image

**Review by magicmatatjahu — DISMISSED** [review]

**Review by aerfio — APPROVED** [review]

**Review by aerfio — APPROVED** [review]

**Review by magicmatatjahu — APPROVED** [review]

**Review by kazydek — APPROVED** [review]
As agreed, approving only for development purposes - texts are not final.

**Pull request merged**